pkgs/applications/kde/kalzium.nix

```diff
@@ -0,0 +1,26 @@
+{ mkDerivation, lib, extra-cmake-modules, kdoctools, ki18n, kio, openbabel, avogadro, qtscript, kparts, kplotting, kunitconversion }:
+
+mkDerivation {
+  name = "kalzium";
+  meta = with lib; {
+    homepage = "https://kde.org/applications/en/utilities/org.kde.kalzium";
+    description = "Kalzium is a program that shows you the Periodic Table of Elements";
+    maintainers = with maintainers; [ freezeboy ];
+    license = licenses.gpl2Plus;
+    platforms = platforms.linux;
+  };
+  nativeBuildInputs = [
+    extra-cmake-modules
+  ];
+  buildInputs = [
+    qtscript
+    #avogadro
```

Comment:
Is this dependency needed?

Reply:
Currently I didn't succeed compiling kalzium with avogadro support (dont really know what it does in fact, just saw the reference in the CMakeLists.txt)

Do you want me to remove the reference ?

Reply:
No, that's OK.